doc/api/string_decoder.md

```diff
@@ -17,23 +23,49 @@ const euro = Buffer.from([0xE2, 0x82, 0xAC]);
 console.log(decoder.write(euro));
 ```
 
-## Class: StringDecoder
+When a `Buffer` instance is written to the `StringDecoder` instance, an
+internal buffer is used to ensure that the decoded string does not contain
+any partial multibyte-characters. These are held in the buffer until the
```

Comment:
tiny nit: `multibyte characters` (and I’d maybe call them `incomplete` instead of `partial` but that’s certainly a question of personal taste)